src/appConfig/appConfig.ts

```diff
@@ -0,0 +1,19 @@
+const config = require('./appConfig.ENV.json'); // tslint:disable-line: no-var-requires
+
+export enum AuthMode {
+    ImplicitFlow = 'implicitFlow',
+    ConnectionString = 'connectionString'
+}
+
+export enum HostMode {
+    Browser = 'browser',
+    Electron = 'electron'
+}
+
+export interface AppConfigInterface {
+    developmentMode: boolean;
+    hostMode: HostMode;
+    authMode: AuthMode;
+}
+
+export const appConfig = config as AppConfigInterface;
```

Comment:
I would prefer to add a helper here to extract the common logic like 'appConfig.hostMode === HostMode.Electron'  or 'appConfig.developmentMode' to provide some convinience. What do you say.